cloudify_agent/installer/script.py

```diff
@@ -62,12 +70,21 @@ def install_script(self, add_ssl_cert=True):
         :return: Install script content
         :rtype: str
         """
+        cert_content = ''
+        if add_ssl_cert:
+            cert_content = self._get_local_cert_content()
+        if self.manager_cert:
+            cert_content = self.manager_cert
+
+        install_agent = False
```

Comment:
Probably this as well (own method). In general, if it's a bit complicated to understand what a block of code does, you just create a new function/method and call it `should_install_agent`, or whatever, to explain what's going on.